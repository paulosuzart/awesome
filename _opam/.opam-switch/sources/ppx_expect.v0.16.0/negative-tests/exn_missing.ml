open! Core

let%expect_test "without trailing output" =
  printf "hello world";
  [%expect "hello world"]
[@@expect.uncaught_exn {| (Failure "hi ho") |}]
;;

let%expect_test "with trailing output" =
  printf "hello world"
[@@expect.uncaught_exn {| (Failure "hi ho") |}]
;;
