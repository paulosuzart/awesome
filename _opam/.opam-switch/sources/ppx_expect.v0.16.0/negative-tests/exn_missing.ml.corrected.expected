open! Core

let%expect_test "without trailing output" =
  printf "hello world";
  [%expect "hello world"]
;;

let%expect_test "with trailing output" =
  printf "hello world";
  [%expect {| hello world |}]
;;
